dashboard/test/ui/features/teacher-dashboard.feature

```diff
@@ -159,7 +159,7 @@ Feature: Using the teacher dashboard
     And I am on "http://code.org/teacher-dashboard?enableExperiments=showProjectThumbnails#/sections"
```

Comment:
Should we also remove this line, as with [here](https://github.com/code-dot-org/code-dot-org/pull/18189/files#diff-20be49f18d84540f961f5eb1eb92b2dbL77)?